aot/src/main/java/com/dylibso/chicory/aot/AotMethodsRefs.java

```diff
@@ -1,20 +1,13 @@
 package com.dylibso.chicory.aot;
 
-import static com.dylibso.chicory.wasm.types.Value.REF_NULL_VALUE;
-
+import com.dylibso.chicory.aot.runtime.AotMethods;
 import com.dylibso.chicory.runtime.Instance;
 import com.dylibso.chicory.runtime.Memory;
-import com.dylibso.chicory.runtime.OpcodeImpl;
 import com.dylibso.chicory.runtime.TableInstance;
-import com.dylibso.chicory.runtime.TrapException;
-import com.dylibso.chicory.runtime.exceptions.WASMRuntimeException;
-import com.dylibso.chicory.wasm.exceptions.ChicoryException;
 import com.dylibso.chicory.wasm.types.Element;
-import com.dylibso.chicory.wasm.types.FunctionType;
 import java.lang.reflect.Method;
-import java.util.List;
 
-public final class AotMethods {
+public final class AotMethodsRefs {
```

Comment:
Let's name this `AotMethodRefs` to avoid the double plural. It also directly matches the English:

* AOT methods
* AOT method references

Reply:
thanks :sweat_smile: